kvm/src/lib.rs

```diff
@@ -20,18 +20,23 @@ mod cap;
 mod ioctl_defs;
 
 use std::fs::File;
+use std::io::Error as IoError;
```

Comment:
Would it make sense to use `std::io::Error` as the same alias in the whole codebase?